hackage-security/hackage-security.cabal

```diff
@@ -29,8 +29,10 @@ category:            Distribution
 homepage:            https://github.com/haskell/hackage-security
 bug-reports:         https://github.com/haskell/hackage-security/issues
 build-type:          Simple
-tested-with:         GHC==8.10.1, GHC==8.8.3, GHC==8.6.5, GHC==8.4.4, GHC==8.2.2, GHC==8.0.2,
-                     GHC==7.10.3, GHC==7.8.4, GHC==7.6.3, GHC==7.4.2
+tested-with:
+  GHC==9.0.2,
+  GHC==8.10.7, GHC==8.8.4, GHC==8.6.5, GHC==8.4.4, GHC==8.2.2, GHC==8.0.2,
```

Comment:
OOI, do you know if this old formatting that we currently have is handled by haskell-ci? I know this one is:

https://github.com/haskell-CI/haskell-ci/blob/master/README.md?plain=1#L40

Reply:
So far, `haskell-ci` has been very robust.  Usually, I even omit the comma (using newlines instead).

Reply:
Btw, this repo has a hand-crafted CI (not auto-generated by haskell-ci).

Reply:
Thank you. I know some variants failed in the past, but perhaps it improved.